storage/container_storage_interface/ephemeral-storage-csi-inline.adoc

```diff
@@ -0,0 +1,16 @@
+[id="ephemeral-storage-csi-inline"]
+= Using CSI inline volumes
+include::modules/common-attributes.adoc[]
+:context: ephemeral-storage-csi-inline
+
+toc::[]
+
+Container Storage Interface (CSI) inline volumes offer a practical way to extend Pods so that node-local integrations can happen. This feature is only available with supported Container Storage Interface (CSI) drivers.
```

Comment:
I can't understand `node-local interations` here.
IIUC, csi inline ephemeral volume is given a way to be able to define pod specs with embedded ephemeral CSI volumes that are created/mounted when the pod is deployed and are deleted when the pod goes away.

Reply:
I like your description better; I changed the intro sentence to read, "Container Storage Interface (CSI) inline volumes allow you to define a Pod spec that creates inline, ephemeral volumes when a Pod is deployed and delete them when a Pod is destroyed."